### PR TITLE
python3Packages.pyfakewebcam: init at 0.1.0

### PR DESCRIPTION
**Pull request by emilytrau**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131617` at d463a1bc run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pyfakewebcam</li>
    <li>python39Packages.pyfakewebcam</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/pyfakewebcam/default.nix:17:0:
```
   |
17 |     description = "A library for writing RGB frames to a fake webcam device on Linux";
   | ^
```
</li>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/development/python-modules/pyfakewebcam/default.nix:1:8:
```
  |
1 | { lib, stdenv, buildPythonPackage, fetchPypi, numpy, opencv4, ... }:
  |        ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/pyfakewebcam/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  propagatedBuildInputs = [ numpy opencv4 ];
+
+  # No tests are available
+  doCheck = false;
+  pythonImportsCheck = [ "pyfakewebcam" ];
+
+  meta = with lib; {
+    description = "A library for writing RGB frames to a fake webcam device on Linux";
+    homepage = "https://github.com/jremmons/pyfakewebcam";
+    license = licenses.gpl3Plus;
```
Comment (fabaff) [inline pkgs/development/python-modules/pyfakewebcam/default.nix:21]: The [License file](https://github.com/jremmons/pyfakewebcam/blob/master/LICENSE.txt) says LGPLv3.

```suggestion
    license = licenses.lgpl3Only;
```

File: pkgs/development/python-modules/pyfakewebcam/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, numpy, opencv4, ... }:
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyfakewebcam/default.nix:1]:
```suggestion
{ lib, buildPythonPackage, fetchPypi, numpy, opencv4 }:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pyfakewebcam/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/pyfakewebcam/default.nix:21]: Please inform the upstream about the licensing mismatch.

Comment (angustrau) [reply pkgs/development/python-modules/pyfakewebcam/default.nix]: Thanks for catching this! Sorry about that

File: pkgs/development/python-modules/pyfakewebcam/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  };
+
+  propagatedBuildInputs = [ numpy opencv4 ];
+
+  # No tests are available
+  doCheck = false;
+  pythonImportsCheck = [ "pyfakewebcam" ];
+
+  meta = with lib; {
+    description = "A library for writing RGB frames to a fake webcam device on Linux";
```
Comment (jonringer) [inline pkgs/development/python-modules/pyfakewebcam/default.nix:19]: if this is only relevant to linux, do you mind adding:
```
    platforms = platforms.linux;
```

File: pkgs/development/python-modules/pyfakewebcam/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/pyfakewebcam/default.nix]: See https://github.com/jremmons/pyfakewebcam/pull/15

**Pull request merged**